packages/genesys-spark-components/src/components/stable/gux-calendar/gux-calendar.tsx

```diff
@@ -547,6 +556,7 @@ export class GuxCalendar {
           <button
             type="button"
             class="gux-right"
+            aria-label={this.i18n('nextMonth')}
```

Comment:
In the `gux-calendar-beta` we have the `month` and `year` appended to this label eg `Next month : June 2025`. Should we do that here aswell ?

Reply:
Maybe I do think that is better UX but I think what is currently there fixes the accessibility issue.
I could create a bug ticket to improve the text to match the beta component.